### PR TITLE
add documentation for known FRR issue 7299 and workaround

### PR DESCRIPTION
**Pull request by jklare**
Signed-off-by: Jan Klare <jan.klare@bisdn.de>

### CODE REVIEW

File: limitations.md [hunk 1]
```diff
@@ -45,3 +45,7 @@ Depending on the switch and the link partner, we have observed the following beh
 - The 25G ports on [AG5648](https://agema.deltaww.com/product-info.php?id=41) only support advertising up to 10G, so the speed will be limited to 10G regardless of the link partner's ability.
 
 In all of these cases forcing the port on the switch to the desired speed works as expected.
+
+## Missing routes for EIGRP with flapping ports
+
+As documented in the currently open upstream FRR issue [#7299](https://github.com/FRRouting/frr/issues/7299), there seems to be an issue when flapping ports during EIGRP session establishment. For now, we recommend the workaround of restarting FRR after all ports are up if this behaviour is observed.
```
Comment (ideaship) [inline limitations.md:51]: s/when flapping ports/when ports are flapping/

Comment (KanjiMonster) [reply limitations.md:51]: s/behaviour/behavior/

And maybe something along 
> ..., routes may get dropped when ports are flapping during EIGRP session establishment.

to describe the issue, and not just state that there seems to be one.

Comment (jklare) [reply limitations.md]: applied as suggested

**Review by KanjiMonster — APPROVED** [review]
lgtm

**Pull request merged**